### DISCUSSION TITLE
Can't export csv

### DISCUSSION THREAD

**Discussion by blangesoci** [dbeaver/dbeaver#35902] [Q&A] [answered]
### Description

When I go to export a csv file there is an error message that no database is selected. 
![Screen Shot 2024-09-27 at 9 33 09 AM](https://github.com/user-attachments/assets/cb24aed1-4e25-4fe2-862b-ff00c22a51b0)

### DBeaver Version

Community Edition Version 24.2.1

### Operating System

macOs Monterey 12.6

### Database and driver

Driver for MySQL 8 and later

### Steps to reproduce

1. I select a database using the use command. 
2. I run a query and the result is displayed. 
3. I right click the result set and select Export 
4. I select Export as CSV and click Next 
5. I click Next to select all default settings 
6. I select a Directory and File name for the export 
7. I click Proceed 
8. The error No Database Select is displayed.

**Comment by serge-rider** [comment, marked as answer]
Most likely it happens because you open separate connection for data export and this new connection doesn't "know" that you executed "use database" in the SQL editor.

Uncheck option "Open new connection" on the Extraction settings page:
<img width="587" alt="image" src="https://github.com/user-attachments/assets/ee0bd9a0-4637-4b5f-9fd6-b1e9d80ab31f">

**Comment by blangesoci** [comment]
that worked. Thank you for the information, very much appreciated.
